app/src/main/aidl/com/tari/android/wallet/service/TariWalletServiceListener.aidl

```diff
@@ -51,4 +51,5 @@ oneway interface TariWalletServiceListener {
 
     void onDiscoveryComplete(in TxId completedTxId, in boolean success);
 
+    void onFreeTariAllocateResult(in boolean success, in String error);
```

Comment:
```suggestion
    void onTestnetTariAllocateResult(in boolean success, in String error);
```

Reply:
@CjS77 This could also be called `onTestnetTariRequestResult`, or even better, two functions: `onTestnetTariRequestSuccess(ful)` and `onTestnetTariRequestError` @jigneshsanghani